index.html

```diff
@@ -310,7 +317,11 @@ <h2 class="title"><a href="./demos/whisper-base/">Whisper Base</a></h2>
           <div class="tags">
             <span>MobileNet</span><span>ResNet</span>
           </div>
-          <div class="tags backend"><a href="./demos/image-classification/?provider=webnn&devicetype=gpu&model=mobilenet-v2&run=5" title="WebNN GPU">GPU</a> <a href="./demos/image-classification/?provider=webnn&devicetype=npu&model=mobilenet-v2&run=5" title="WebNN NPU">NPU</a></div>
+          <div class="tags backend"><a
+              href="./demos/image-classification/?provider=webnn&devicetype=gpu&model=mobilenet-v2&run=5"
+              title="WebNN GPU">GPU</a> <a
+              href="./demos/image-classification/?provider=webnn&devicetype=npu&model=mobilenet-v2&run=5"
+              title="WebNN NPU">NPU</a></div>
```

Comment:
```suggestion
          <div class="tags backend"><a
            href="./demos/image-classification/?provider=webnn&devicetype=gpu&model=mobilenet-v2&run=5"
            title="WebNN GPU">GPU</a> <a
            href="./demos/image-classification/?provider=webnn&devicetype=npu&model=mobilenet-v2&run=5"
            title="WebNN NPU">NPU</a></div>
```
nit, consistent indent within the same file (install.html uses 4, which is fine, and index.html uses 2, which is fine). Line 214 uses 2.

Reply:
Fixed, thanks!